internals/overlord/servstate/manager.go

```diff
@@ -59,16 +60,18 @@ func (e *LabelExists) Error() string {
 	return fmt.Sprintf("layer %q already exists", e.Label)
 }
 
-func NewManager(s *state.State, runner *state.TaskRunner, pebbleDir string, serviceOutput io.Writer, restarter Restarter, logMgr LogManager) (*ServiceManager, error) {
+func NewManager(s *state.State, runner *state.TaskRunner, pebbleDir string, pebbleImportDirs []string,
```

Comment:
This is now a 7-param function ... time to use an Options struct? (It's in `internals` so we can change the API.)

Reply:
Great idea, I've gone ahead and simplified a lot of this.